### PR TITLE
fix: fix checkbox UI issue in invite user page

### LINKED ISSUE #4545: Design Issue in Checkbox of Invite a new user page
Checkbox design in "Invite a new user page" doesn't look good.
It shows as the following screenshot:
image

Page Navigation:
Settings > Users > Invite Someone

URL: /settings/users/create

### PR DESCRIPTION
**Pull request by kavanpancholi**
This PR fixes Checkbox UI issue in "Invite a new user page"
Issue: #4545 

Page Navigation:
Settings > Users > Invite Someone

URL: /settings/users/create

Before:
image

After:
image

#### Before submitting the PR
- [x] Read the [CONTRIBUTING document](https://github.com/monicahq/monica/blob/master/CONTRIBUTING.md) before submitting your PR.
- [x] If the PR is related to an issue or fix one, don't forget to indicate it.
- [x] Create your PR as draft if it is not final yet. Mark it as ready... when it’s ready. Otherwise the PR will be considered complete and rejected if it's not working.

### General checks
- [x] Make sure that the change you propose is the smallest possible.
- [x] The name of the PR should follow the [conventional commits guideline](https://github.com/monicahq/monica/blob/master/docs/contribute/readme.md#conventional-commits) that the project follows.

### Front-end changes
- [x] If you change the UI, make sure to ask repositories administrators first about your changes by pinging djaiss or asbiin in this PR.
- [x] Screenshots are included if the PR changes the UI.
- [x] Front-end tests have been written with Cypress.

#### Backend/models changes
- [x] The API has been updated.
- [x] API's documentation has been added by submitting a pull request in the [marketing website repository](https://github.com/monicahq/marketing_site/pulls).
- [x] Tests have been added for the new code.
- [x] If you change a model, make sure the SetupTest file is updated. We need seeders to develop locally and generate fake data.

#### If the code changes the SQL schema
- [x] Make sure exporting account data as SQL is still working.
- [x] Make sure your changes do not break importing data with `vCard` and `.csv` files.
- [x] Make sure account reset and deletion still work.

#### Other tasks
- [x] [CHANGELOG](https://github.com/monicahq/monica/blob/master/CHANGELOG.md) entry added, if necessary, under `UNRELEASED`.
- [x] [CONTRIBUTORS](https://github.com/monicahq/monica/blob/master/CONTRIBUTORS) entry added, if necessary.
- [x] If it's relevant and worth mentioning, create a changelog entry for this change. The changelog entry will appear inside the UI for all users to see. To know if your change is worth the creation of a changelog entry, [read the documentation](https://github.com/monicahq/monica/blob/master/docs/administrators/tips.md#when-is-it-relevant-to-create-a-changelog-entry).
- [x] Don't forget to [ask for a free account](mailto:regis@monicahq.com) on https://monicahq.com as anyone who contributes can request a free account.

### CODE REVIEW

**Comment by kavanpancholi** [comment]
Hey, @asbiin Can you please review this PR?
And if you feel like merging it, can you please add "hacktoberfest-accepted" label to it?
Thank you. :+1: 
Cheers.

File: resources/sass/app-ltr.scss [hunk 1]
```diff
@@ -274,6 +274,21 @@ ul {
...
+  .state label{
+    text-indent: 0;
+    padding-left: 2rem;
+    line-height: 1.4em;
+    &:after, &:before {
+      top: 0;
+    }
+  }
+}
+
```
Comment (asbiin) [inline resources/sass/app-ltr.scss:291]: That's great!

However, we will have to adapt it so it works for RTL direction too:
```suggestion
.pretty {
  white-space: inherit;
  &.form-check-input {
    @if $htmldir == ltr {
      margin-left: 0;
    } @else {
      margin-right: 0;
    }
  }
  .state label{
    text-indent: 0;
    @if $htmldir == ltr {
      padding-left: 2rem;
    } @else {
      padding-right: 2rem;
    }
    line-height: 1.4em;
    &:after, &:before {
      top: 0;
    }
  }
}

```

**Review by asbiin — CHANGES_REQUESTED** [review]
Really nice! Thanks.
See my comment above

File: resources/sass/app-ltr.scss (hunk 1 above)
Comment (kavanpancholi) [reply resources/sass/app-ltr.scss]: Thank you @asbiin for the suggestion.
I've made the changes.

**Comment by kavanpancholi** [comment]
Hey @asbiin , I'd appreciate if you can add label "hacktoberfest-accepted" before merging it
It would help me in hacktoberfest.
Thanks.

**Comment by asbiin** [comment]
@kavanpancholi I will merge it, so it does not need the label at all.

**Comment by asbiin** [comment]
Thank you @kavanpancholi

**Pull request merged**